src/FirstSteps/RPG/Heroes/Boss.cs

```diff
@@ -17,17 +18,18 @@ private Boss()
 
         public static Boss CreateDiablo()
         {
-            return new Boss { _name = "Diablo", Health = 50, Strength = 5, Damage = 6 };
+            return new Boss { Name = "Diablo", Health = 40, Strength = 6, Damage = 6 };
         }
 
         public int DealDamage()
-        {
-            return Damage + Strength;
+        {           
+            return Damage;       
+            // i tryed make a logic with reduced damage here but not working. 
```

Comment:
The place is good. You just need to make a calculation. As a tip, you must store `MaxHealth` somewhere and use it in calculations every time.